client/src/services/hotkeyManager.ts

```diff
@@ -0,0 +1,126 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+import { isDesktop, isLinux, isMacOS, isMobile, isWeb, isWindows } from '@/parsec/environment';
+
+export const HotkeyManagerKey = 'hotkey';
+
+export enum Modifiers {
+  Ctrl = 1 << 0,
+  Alt = 1 << 1,
+  Shift = 1 << 2,
+}
+
+export enum Platforms {
+  Windows = 0,
```

Comment:
Should start at `1 << 0` (or `1`). It's impossible to check if `Platforms.Windows` is set otherwise. If you want to add a 0 value, it should be something like `NoPlatforms`. I don't know whether it will be used though.